stories/RadioButtonGroup-stories.tsx

```diff
@@ -0,0 +1,52 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { RadioButtonGroup } from '@sonarsource/echoes-react';
+
+const meta = {
+  component: RadioButtonGroup,
+  title: 'RadioButtonGroup',
+} satisfies Meta<typeof RadioButtonGroup>;
+
+export default meta;
+
+type Story = StoryObj<typeof RadioButtonGroup>;
```

Comment:
Seems like this can be `typeof meta` when using the satisfies keyword, makes for less things to replace when copy pasting stories :P